### DISCUSSION TITLE
Update github.com/theupdateframework/go-tuf library as it has HIGH-severity vulnerability

### DISCUSSION THREAD

**Discussion by sinke237** [aquasecurity/trivy#7713] [Bugs]
### Question

Before you freak out i have read the [contributing.md](https://github.com/aquasecurity/trivy/blob/672e886aed152ae0f09a16941706746f3053ca94/CONTRIBUTING.md)

Unfortunately it has not given me enough info about how I should start.
I have seen something that will me particularly glad if I am the one who fixed it(see the attachment)
![Screenshot from 2024-10-10 17-40-37](https://github.com/user-attachments/assets/40a7c4ad-3ca5-426f-ad78-06dff67fe513)

I will love to talk with someone who can hint me how to go about this.

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by itaysk** [comment]
Hi, if you have something you would like to work on, please start a Discussions with the appropriate category (Bug/Idea) and explain what it is and what you intend to do about it. We might discuss the validity or implementation details and if theres concensus we will you can start working on it, and eventually submit a Pull Request. We will review the Pull Request and possibly provide feedback, and finally merge it.

**Reply by sinke237** [reply]
Hi @itaysk ,
Thank you for your attention to this. It's encouraging to see your interest.

My intention is to:

Use the aquasec/trivy Docker image to perform vulnerability testing.
I began by scanning the image itself and discovered a HIGH-severity vulnerability.

Given that this is a security concern, it's not feasible to use this image in my pipeline for scanning.

While I find trivy incredibly helpful, I've considered contributing to the project.

I'd like to start by:

Updating the library with the HIGH-severity vulnerability (as seen in the attachment).
I would appreciate your validation of this finding and guidance on how I can continue to contribute effectively to this excellent project.
![Screenshot from 2024-10-10 17-40-37](https://github.com/user-attachments/assets/a5f8b98d-f8b0-4e7b-aea2-eb61d12a4487)

**Reply by itaysk** [reply]
it seems this is an indirect dependency and we need to wait to rekor for updating it. you can try to contribute it there if you want
